Framework/MDAlgorithms/src/FindPeaksMD.cpp

```diff
@@ -399,7 +451,7 @@ void FindPeaksMD::findPeaks(typename MDEventWorkspace<MDE, nd>::sptr ws) {
 
       // The "bin count" used will be the box density or the number of events in
       // the box
-      double binCount = box->getSignalNormalized() * m_densityScaleFactor;
+      double binCount = box->getSignalByNEvents() * m_densityScaleFactor;
```

Comment:
Good spot :+1:. Using `getSignalNormalized` makes much more sense here. At this point in the code we have already found our peaks.